### ISSUE TITLE
Error: Error: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by lewis617** [naeramarth7/hexo-command-gzip#1]
```
/Users/liuyiqi/code/lewis617.github.io/node_modules/gzipme/lib/gzipme.js:50
    throw new Error(e);
    ^

Error: Error: ENOENT: no such file or directory, rename '/Users/liuyiqi/code/lewis617.github.io/public/archives/2017/10/index.html.gz' -> '/Users/liuyiqi/code/lewis617.github.io/public/archives/2017/10/index.html'
```
I just install this plugin and run `hexo gzip`, and this error was thrown out.

**Comment by naeramarth7** [comment]
Which version of `gzipme` have you installed?  
This command relies on `"gzipme": "0.0.9"` as stated in the `package.json` - looks like you have a newer version installed?

* see if another package uses it with `yarn why`
  * if not, you may have installed it by yourself - remove it and reinstall `hexo-command-gzip`
  * if so, see if the other package works with `0.0.9` as well - if not, let me know

**Comment by lewis617** [comment]
```
$ yarn add hexo-command-gzip

yarn add v1.2.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
success Saved 5 new dependencies.
├─ coffee-script@1.12.7
├─ foreachasync@3.0.0
├─ gzipme@0.0.9
├─ hexo-command-gzip@0.0.1
└─ walk@2.3.9
✨  Done in 14.09s.
```
```
$ yarn why gzipme
yarn why v1.2.1
[1/4] 🤔  Why do we have the module "gzipme"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
info This module exists because "hexo-command-gzip" depends on it.
info Disk size without dependencies: "88kB"
info Disk size with unique dependencies: "144kB"
info Disk size with transitive dependencies: "144kB"
info Number of shared dependencies: 1
✨  Done in 0.55s.
```

**Comment by naeramarth7** [comment]
Tracked it down to an issue in `gzipme` mentioned here caio-ribeiro-pereira/gzipme#5.

**Comment by naeramarth7** [comment]
This should be fixed in [v0.1.0](https://github.com/naeramarth7/hexo-command-gzip/releases/tag/v0.1.0) which also removes the `gzipme`.

Thanks for reporting!
